bots/image-refresh

```diff
@@ -133,17 +150,35 @@ def run(image, verbose=False, **kwargs):
 
     branch = task.branch(image, "images: Update {0} image".format(image), pathspec="bots/images", **kwargs)
     if branch:
-        pull = task.pull(branch, **kwargs)
 
-        # we want to validate all image refreshes with external projects
-        task.label(pull, ["test-external"])
+        pull = task.pull(branch, run_tests=False, **kwargs)
 
         # Trigger this pull request
         api = github.GitHub()
+        old_head = pull["head"]["sha"]
         head = api.get("pulls/{}".format(pull["number"]))["head"]["sha"]
+        retry = 3
+        while head == old_head and retry:
+            time.sleep(3)
```

Comment:
How sure are you that the API will always be consistent after 6+ε seconds? IMHO it can't hurt to retry more often here, maybe 20 for a nice round minute?

Also, style nitpick (completely optional): I find that a retry loop is more concise and robust like this:
```py
for retry in seq(20):
    if abort_condition:
         break
    time.sleep(3)
else:
    # timeout: panic, abort, kittens die
```